source/index.html.md

```diff
@@ -1340,6 +1340,176 @@ Delete an existing Organization's Address
 | -------------------------------- | ----- | ----------------------------------| ---------------------------------------------- |
 | /OrganizationAddresses/{addr_id} | DELETE  | Not required | Number of deleted objects |
 
+-----
+
+## Products Endpoint
+The Products represent line items that might be referenced in the documents you send Shipamax to processes. The product reference data is used to improve the accuracy of the parsing process, making sure the most likely product is selected. 
+Each Product must be unique across its product code, importer and supplier.
+
+### Attributes
+
+| Attribute                               |  Description                                                      |
+| --------------------------------------- | ----------------------------------------------------------------- |
+| id                 | Unique identifier of the Product within the Shipamax system |
+| productCode                               | The code of the Product within your own system           |
+| owner                       | Unique identifier of the owner of the product within your own system     |
+| supplier                       | Unique identifier of the supplier of the product within your own system     |
+| description                       | Description of the product  |
+| unitType                       | The unit the product is quantified by   |
+| tariff                       | The tariff for the product   |
+| origin                       | The origin country of the product |
+
+
+
+### POST
+Create a new Product
+
+| Endpoint                         | Verb  | Body                              | Response                                       |
+| -------------------------------- | ----- | ----------------------------------| ---------------------------------------------- |
+| /Products                   | POST  | Product's details in JSON    |  The new Product object in JSON           |
+
+> **Body structure for POST Products request:**
+
+```json
+{
+    "productCode": string,
+    "owner": string,
+    "supplier": string,
+    "description": string,
+    "unitType": string,
+    "tariff": string,
+    "origin": string
+}
+```
+
+> **Example:** POST Product request body
+
+```json
+{
+    "productCode": "CODEABC",
+    "owner": "TRRRRFF",
+    "supplier": "BRRRRGG",
+    "description": "Description of product",
+    "unitType": "PKG",
+    "tariff": "123456",
+    "origin": "GB"
+}
+```
+
+> **Example:** POST Product response
+
+```json
+{
+  "id": 35,
+  "productCode": "CODEABC",
+  "owner": "TRRRRFF",
+  "supplier": "BRRRRGG",
+  "description": "Description of product",
+  "unitType": "PKG",
+  "tariff": "123456",
+  "origin": "GB"
+}
+```
+
+### GET (specific Product)
+Retrieve details of a an existing Product
```

Comment:
I would make it:

Retrieve details of a an existing Product reference by using the product code from your system.
If there are multiple product with the same code they will all be included in the response